python/processors/Stop0l_postproc.py

```diff
@@ -59,16 +129,32 @@ def main(args):
 
     #~~~~~ For MC ~~~~~
     if not isdata:
-        pufile = "%s/src/PhysicsTools/NanoSUSYTools/data/pileup/%s" % (os.environ['CMSSW_BASE'], DataDepInputs[args.era]["pileup"])
+        pufile = "%s/src/PhysicsTools/NanoSUSYTools/data/pileup/%s" % (os.environ['CMSSW_BASE'], DataDepInputs["MC"][args.era]["pileup"])
         mods += [
-            # jecUncertProducer(DataDepInputs[args.era]["JECU"]),
-            PDFUncertiantyProducer(isdata),
+            jecUncertProducer(DataDepInputs["MC"][args.era]["JECMC"]),
+            jetmetUncertaintiesProducer(args.era, DataDepInputs["MC"][args.era]["JECMC"], jerTag=DataDepInputs["MC"][args.era]["JERMC"], redoJEC=DataDepInputs["MC"][args.era]["redoJEC"], doSmearing=False),
+#            PDFUncertiantyProducer(isdata),
             # lepSFProducer(args.era),
             puWeightProducer("auto", pufile, "pu_mc","pileup", verbose=False),
+            btagSFProducer(era=args.era, algo="deepcsv"),
+            BtagSFWeightProducer("allInOne_bTagEff_deepCSVb_med.root", args.sampleName, DeepCSVMediumWP[args.era]),
             # statusFlag 0x2100 corresponds to "isLastCopy and fromHardProcess"
             # statusFlag 0x2080 corresponds to "IsLastCopy and isHardProcess"
-            GenPartFilter(statusFlags = [0x2100, 0x2080]),
+            GenPartFilter(statusFlags = [0x2100, 0x2080, 0x2000], pdgIds = [0, 0, 22], statuses = [0, 0, 1]),
+            TopTaggerProducer(recalculateFromRawInputs=True,                   AK4JetInputs=("Jet_pt",              "Jet_eta", "Jet_phi", "Jet_mass"),              topDiscCut=0.6),
+            TopTaggerProducer(recalculateFromRawInputs=True, suffix="JESUp",   AK4JetInputs=("Jet_pt_jesTotalUp",   "Jet_eta", "Jet_phi", "Jet_mass_jesTotalUp"),   topDiscCut=0.6),
+            TopTaggerProducer(recalculateFromRawInputs=True, suffix="JESDown", AK4JetInputs=("Jet_pt_jesTotalDown", "Jet_eta", "Jet_phi", "Jet_mass_jesTotalDown"), topDiscCut=0.6),
+        ]
+    else:
+        if DataDepInputs["Data"][args.era + args.dataEra]["redoJEC"]:
```

Comment:
so the dataEra will be just B/C/D/E ...?  Is this supported from the current cfg file?

Reply:
This is something we need to discuss with Scarlet and Anna.  Last night I learned the data names are not uniformly formatted between 2016/17/18 in the cfg files.  Once this is fixed so they are all separated by period, we should have no issue.

Reply:
Ok, I asked @ahenckel to talk with you to figure out what you need and how we can propagate from cfg file via SubmitLPC.py to here.